src/main/kotlin/no/nav/familie/ef/sak/brev/VedtaksbrevService.kt

```diff
@@ -46,6 +81,22 @@ class VedtaksbrevService(private val brevClient: BrevClient,
         return beslutterPdf.bytes
     }
 
+    fun lagFritekstbrev(frittståendeBrevDto: VedtaksbrevFritekstDto): ByteArray {
+        val ident = behandlingService.hentAktivIdent(frittståendeBrevDto.behandlingId)
+        val navn = personopplysningerService.hentGjeldeneNavn(listOf(ident))
+        val request = FrittståendeBrevRequestDto(overskrift = frittståendeBrevDto.overskrift,
+                                                 avsnitt = frittståendeBrevDto.avsnitt,
+                                                 personIdent = ident,
+                                                 navn = navn[ident]!!,
+                                                 brevdato = LocalDate.now())
+        val vedtaksbrev = lagreEllerOppdaterVedtaksbrev(behandlingId = frittståendeBrevDto.behandlingId,
+                                                        brevrequest = objectMapper.writeValueAsString(request),
+                                                        brevmal = FRITEKST,
+                                                        saksbehandlersignatur = SikkerhetContext.hentSaksbehandlerNavn(true))
+
+        return brevClient.genererBrev(vedtaksbrev = vedtaksbrev)
+    }
```

Comment:
Fint om vi sjekker at behandlingen er i riktig tilstand ved lagring av brev. Se `lagSaksbehandlerBrev` for inspirasjon :) 

```suggestion
    fun lagFritekstbrev(frittståendeBrevDto: VedtaksbrevFritekstDto): ByteArray {
       val behandling = behandlingService.hentBehandling(frittståendeBrevDto.behandlingId)
        if (behandling.status.behandlingErLåstForVidereRedigering()) {
            throw Feil("Behandling er i feil steg=${behandling.steg} status=${behandling.status}",
                       httpStatus = HttpStatus.BAD_REQUEST)
        }
        val ident = behandlingService.hentAktivIdent(frittståendeBrevDto.behandlingId)
        val navn = personopplysningerService.hentGjeldeneNavn(listOf(ident))
        val request = FrittståendeBrevRequestDto(overskrift = frittståendeBrevDto.overskrift,
                                                 avsnitt = frittståendeBrevDto.avsnitt,
                                                 personIdent = ident,
                                                 navn = navn[ident]!!,
                                                 brevdato = LocalDate.now())
        val vedtaksbrev = lagreEllerOppdaterVedtaksbrev(behandlingId = frittståendeBrevDto.behandlingId,
                                                        brevrequest = objectMapper.writeValueAsString(request),
                                                        brevmal = FRITEKST,
                                                        saksbehandlersignatur = SikkerhetContext.hentSaksbehandlerNavn(true))

        return brevClient.genererBrev(vedtaksbrev = vedtaksbrev)
    }
```

Reply:
Og dette kan sikkert skilles ut som en privat felles funksjon som `lagSaksbehanlderBrev` og `lagFritektsbrev`, f.eks. `validerBehandlingErRedigerbar`

Reply:
Fikset i 2d7cca8